### PR TITLE
PublisherId not publisher name and updated test

### PR DESCRIPTION
**Pull request by svenngjeto**
Using publisherId ('Forlag id' as NSD calls it) and not publisher name('Utgiver' as NSD calls it) for identifying publisher of a journal. Test is updated to actuall test this - was an hidden exception within test...

### CODE REVIEW

File: src/tests/unit/handler.test.js [hunk 1]
```diff
@@ -426,31 +426,34 @@ describe('Handler returns application/ld+json with deployment path as part of id
...
+      expect(hit.publisherId).to.contain(publisherId)
     })
   })
   it(`returns 200 OK and deployment path as part of id for ${testEvent.path} when it does not have publisher `, async () => {
     const publisherId = [null, undefined, '']
-    publisherId.forEach(async (publisherId) => {
-      const returnValue = journalRemoteResponseDataWithoutPublisher
-      returnValue['forlag id'] = publisherId
-      nsdMockReturns(httpStatus.OK, returnValue)
+    for (const publisherId1 of publisherId) {
```
Comment (axthosarouris) [inline src/tests/unit/handler.test.js:445]: rename the variables. or do not use a for loop

File: src/tests/unit/handler.test.js [hunk 2]
```diff
@@ -426,31 +426,34 @@ describe('Handler returns application/ld+json with deployment path as part of id
...
+      const returnValue = JSON.parse(journalRemoteResponseDataWithoutPublisher)
+      returnValue['forlag id'] = publisherId1
```
Comment (axthosarouris) [inline src/tests/unit/handler.test.js:447]: rename publisherId1

**Review by axthosarouris — CHANGES_REQUESTED** [review]
Fix the comments

File: src/tests/unit/handler.test.js (hunk 1 above)
Comment (svenngjeto) [reply src/tests/unit/handler.test.js]: Ups, IDE optimized forEach... Rolled back to 'forEach'

File: src/tests/unit/handler.test.js (hunk 2 above)
Comment (svenngjeto) [reply src/tests/unit/handler.test.js]: Renamed variables

**Review by axthosarouris — APPROVED** [review]

**Pull request merged**